src/pages/AppNewPassword.vue

```diff
@@ -197,6 +295,15 @@ export default {
       }
     }
 
+    function skipPasswordCreation() {
+      localStorage.setItem("skipPassword", true);
```

Comment:
[The `Storage.setItem` method](https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem) only accepts DOMString i.e. String as the second argument. When you set it to true, it's actually storing "true" in LS. The only reason the `getItem` logic also works is because of implicit coercion. It's not a good idea to rely on this.

I recommend explicitly storing "true" or "1" and comparing it like `getItem("skipPassword") !== 'true'`.

![image](https://user-images.githubusercontent.com/29047033/147804735-02dcff10-da2c-438c-817f-0327b3d476e5.png)

Reply:
Yeah this makes sense.